app/facades/foods_facade.rb

```diff
@@ -0,0 +1,17 @@
+class FoodsFacade
+
+  def self.search_foods(query, page_size = 10)
+    json = FoodService.search(query, page_size)
+    foods = json[:foods].map do |food_data|
+      data = {}
+      data[:code] = food_data[:gtinUpc]
+      data[:description] = food_data[:lowercaseDescription]
+      data[:brand] = food_data[:brandOwner]
+      data[:ingredients] = food_data[:ingredients]
+      Food.new(data)
+    end
+    total_hits = json[:totalHits]
+    return [foods, total_hits]
+  end
```

Comment:
Really nice work on this method.
- a default argument works well here to make the API call in FoodService more dynamic and flexible.
- since you're returning both `json[:totalHits]` and 10 poros created from `json[:foods]`, there's no need for an extraneous API call.